tests/test_localization.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+import unittest
+import sys
+import os
+import mssqlcli.i18n as i18n
+import shutil
+
+
+class LocalizationTests(unittest.TestCase):
+
+    def test_product(self):
+        mo_config = {
+            'extraction_target': i18n.MODULE_ROOT_DIR,
+            'lang_name': 'ko_KR',
+            'domain': i18n.DOMAIN,
+            'trans_mappings': {
+                u'Goodbye!': u'안녕히가세요!',
+            }
+        }
+        i18n.generate_mo(mo_config)
+
+        i18n.LANGUAGES = ['ko']
```

Comment:
Let's check in the .mo file with localized 'goodbye!' string. This way it's picked by the unit tests. Once we have the lcl->mo translation process in place, we'll remove this .mo file since they will be generated as part of the build process.